packages/firestore/lib/modular/query.js

```diff
@@ -130,27 +130,25 @@ export function startAfter(...docOrFields) {
 }
 
 /**
- * @param {number | string | boolean | null} value
- * @param {string?} key
- * @returns {QueryConstraint}
+ * Creates a QueryEndAtConstraint that modifies the result set to end at the provided fields relative to the order of the query.
+ * The order of the field values must match the order of the order by clauses of the query.
+ *
+ * @param {*} fieldValues
  */
-export function endAt(value, key) {
-  if (!key) {
-    return new QueryConstraint('endAt', value);
-  }
-  return new QueryConstraint('endAt', value, key);
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function endAt(fieldValues) {
+  throw new Error('endAt is not implemented');
 }
```

Comment:
This needs to be revised to match the firebase-js-sdk:

I think it should literally be: 

```js
endAt(...fieldValues) {
  return new QueryConstraint('endAt', ...fieldValues);
}
```